<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMaterial</class>
 <widget class="QWidget" name="AddMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>МАТЕРІАЛ</string>
  </property>
  <widget class="QLabel" name="ExpiritDateLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Термін придатності</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Weight">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MadeDateLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата виготовлення</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddMaterialButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Додати</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdMaterial">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CountLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>159</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Кількість</string>
   </property>
  </widget>
  <widget class="QLabel" name="WeightLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Вага</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="IdMaterialLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID матеріалу</string>
   </property>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Назва</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Count">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="QualityLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>189</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Якість</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Quality">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DescriptionLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>219</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Опис</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Description">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="MadeDate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="ExpiritDate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
